src/api/app/models/notification.rb

```diff
@@ -36,6 +36,8 @@ class Notification < ApplicationRecord
   scope :for_comments, -> { where(notifiable_type: 'Comment') }
   scope :for_requests, -> { where(notifiable_type: 'BsRequest') }
   scope :for_reviews, -> { where(event_type: 'Event::ReviewWanted') }
+  scope :for_added_user_to_groups, -> { where(notifiable_type: 'GroupsUser', event_type: 'Event::AddedUserToGroup') }
+  scope :for_removed_user_from_groups, -> { where(notifiable_type: 'GroupsUser', event_type: 'Event::RemovedUserFromGroup') }
```

Comment:
I doubt people want to filter for either. I would only make a `Group` scope/filter

Reply:
Unified into one.